cmd/cloudx/project/get_oauth2_config.go

```diff
@@ -0,0 +1,55 @@
+package project
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/ory/cli/cmd/cloudx/client"
+	"github.com/ory/x/cmdx"
+)
+
+func NewGetOAuth2ConfigCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "oauth2-config <project-id>",
+		Aliases: []string{"oc", "oauth2-config"},
+		Args:    cobra.ExactArgs(1),
+		Short:   "Get an Ory Cloud project's OAuth2 configuration",
+		Long:    "You can use this command to render Ory OAuth2 Configuration as well.",
```

Comment:
The `Long` description as written here ("you can also ...") only makes sense if the reader read the `Short` description first. But Cobra doesn't do that. It shows the `Short` description in the overview of all commands and the `Long` description when showing help for a specific command. I therefore suggest to copy the short description into the long one:

```suggestion
		Short:   "Get an Ory Cloud project's OAuth2 configuration",
		Long:    "Get an Ory Cloud project's OAuth2 configuration. You can use this command to render Ory OAuth2 Configuration as well.",
```

Reply:
Nice find, thank you!

Reply:
I think we do this incorrectly for all commands. Would you be open to make a PR to fix all of the help commands? :)

Reply:
yes, created #199 to track this